webapp/channels/src/components/widgets/inputs/users_emails_input.tsx

```diff
@@ -214,7 +217,7 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
         </>
     );
 
-    Input = (props: InputProps) => {
+    Input = (props: InputProps<EmailInvite | UserProfile, boolean>) => {
```

Comment:
You can simplify quite a lot of this by passing `true` instead of `boolean` because we're always passing `isMulti={true}` into the ReactSelect below